packages/synapse-react-client/src/lib/containers/AccessRequirementListV2/AccessApprovalCheckMarkV2.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import {
+  AccessTimeFilled,
+  CheckTwoTone,
+  LockTwoTone,
+} from '@mui/icons-material'
+import Skeleton from '@mui/material/Skeleton'
+import ConditionalWrapper from '../utils/ConditionalWrapper'
+import { Box, SxProps, useTheme } from '@mui/material'
+
+export enum RequirementItemState {
+  COMPLETE = 'COMPLETE',
+  PENDING = 'PENDING',
+  LOCKED = 'LOCKED',
+  LOADING = 'LOADING',
+}
+
+export type CheckMarkProps = {
+  state: RequirementItemState
+  sx?: SxProps
+}
+
+const iconSx: SxProps = {
+  width: '20px',
+}
+
+export default function AccessApprovalCheckMarkV2({
+  state,
+  sx,
+}: CheckMarkProps) {
+  const theme = useTheme()
+  let backgroundColor: string | undefined = undefined
+  let icon: React.ReactNode = <></>
+
+  switch (state) {
+    case RequirementItemState.COMPLETE:
+      backgroundColor = theme.palette.success.main
+      icon = <CheckTwoTone sx={iconSx} />
+      break
+    case RequirementItemState.PENDING:
+      // TODO: Use theme to provide this color
+      // Should always be Sapphire 500, not the primary color of the current app
+      backgroundColor = '#395979'
+      icon = <AccessTimeFilled sx={iconSx} />
+      break
+    case RequirementItemState.LOCKED:
+      // TODO: Use theme to provide this color
+      backgroundColor = '#FF9B00'
```

Comment:
This is orange but is not the same as our "warning" orange. Not yet sure if/how we should introduce colors into the theme that might be one-offs.